Ignore eml files when processing extracted reports

Given `input.delete = no`

After eml files are processed and attachments have been extracted, the convert/parse phase attempts to reprocess the eml files as reports and logs an error:

```
[ERROR] files: mimetype text/plain; charset=utf-8 not supported in file dmarc_files/foo.eml, skip
```
We should ignore the eml files in this phase.

I plan to submit a PR shortly to resolve this issue.

--- a/cmd/dmarc-report-converter/files.go
+++ b/cmd/dmarc-report-converter/files.go
@@ -1,6 +1,7 @@
 package main
 
 import (
+	"io/fs"
 	"log"
 	"os"
 	"path/filepath"
@@ -85,7 +86,24 @@ func (c *filesConverter) find() error {
 		}
 	}
 
-	files, err := filepath.Glob(filepath.Join(c.cfg.Input.Dir, "*.*"))
+	// Walk Input.Dir for a list of files to process, skipping eml files.
+	var files []string
+	err = filepath.Walk(c.cfg.Input.Dir, func(path string, info fs.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if info.IsDir() {
+			// Process the root dir but don't recurse
+			if path != c.cfg.Input.Dir {
+				return filepath.SkipDir
+			}
+		} else if filepath.Ext(path) != ".eml" {
+			files = append(files, path)
+		}
+
+		return nil
+	})
 	if err != nil {
 		return err
 	}

